scripts/haproxy/haproxy_osver7.cfg

```diff
@@ -122,6 +122,11 @@ frontend main
     acl s3authbackendacl dst_port 9080
     use_backend s3-auth if s3authbackendacl
 
+    # Add a Request ID to request header and haproxy log
+    unique-id-format        %{+X}o\ %ci:%cp_%fi:%fp_%Ts_%rt:%pid
```

Comment:
we can drop frontend ip and port, as it will always be same on a node

Reply:
Will do